zenhappens/src/services.html

```diff
@@ -0,0 +1,371 @@
+---
+layout: layouts/default.njk
+title: Services | Zen Happens 
+---
+
+    {% include "_header.html" %}
+
+     <section class="services__hero">
+        <p class="eyebrow--onimage">Zen Happens in Lauraville</p>
+        <h3 class="landing__text">Services and Pricing</h3>
+    </section>
+    
+    <main class="services__main">
+    <section class="container">
+        <h2 class="price-list__title">Custom Massage</h2>
+        <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
+            <td colspan="2" class="u-sansserif"><strong>Regular Rates</strong></td>
+            
+        </tr>
+        <tr>
+            <td>30 minute massage</td>
+            <td class="table__price">$45</td> 
+        </tr>
+        <tr>
+            <td>45 minute massage</td>
+            <td class="table__price">$65</td> 
+        </tr>
+        <tr>
+            <td>60 minute massage</td>
+            <td class="table__price">$85</td> 
+        </tr>
+        <tr>
+            <td class="table__add-detail">plus full body scrub</td>
+            <td class="table__price">+$100</td> 
+        </tr>
+        <tr>
+            <td>75 minute pre-natal massage</td>
+            <td class="table__price">$110</td> 
+        </tr>
+        <tr>
+            <td>90 minute massage</td>
+            <td class="table__price">$125</td> 
+        </tr>
+        </table>
+
+        <table class="table-style">
+        <tr>
+            <th colspan="2" class="u-sansserif"><strong>Introductory Rates (For First Time Clients)</strong></th>
```

Comment:
This `th` also needs to have its parent `tr` wrapped in a `thead` (see example above). This is consistent, so I won't call out everywhere this is happening and leave it to you to suss out.